io.github.bcpierce00.unison.yml

```diff
@@ -0,0 +1,85 @@
+app-id: "io.github.bcpierce00.unison"
+runtime: "org.freedesktop.Platform"
+runtime-version: "24.08"
+sdk: "org.freedesktop.Sdk"
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.ocaml"
+
+command: "unison-gui"
+finish-args:
+  # must be able to sync anything
+  - "--filesystem=home"
+  # user may want to sync flatpak apps' data/config, is not included in "=home"
+  - "--filesystem=~/.var/app"
```

Comment:
this needs an exception https://docs.flathub.org/docs/for-app-authors/linter#exceptions

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-flatpak-appdata-folder-access

Reply:
oops sorry forgot to rerun the linter, which previously complained only about transient screenshot issues...

please see https://github.com/flathub-infra/flatpak-builder-lint/pull/643